Asterix/Example_param_file.ini

```diff
@@ -32,7 +32,7 @@ onbench=True
     diam_pup_in_m = 8.3e-3
 
     # pupil diameter (in pixels)
-    diam_pup_in_pix = 125
+    diam_pup_in_pix = 100
```

Comment:
Is there a particular reason for this choice? I just put in 126 when I tested it but I guess 100 px is just as good.

Reply:
Yes actually if you are using the Roman pupil it open the 500x500 .fits file and then rebin it. It throws an error if diam_pup_in_pix is not an even divisor (100, 250, 500) (and before this PR, 125 was also a good option). For a clear pupil, this is not important but it's probably better to put 100 at the default option for this reason, you can change it if you want to 126 is you want